### DISCUSSION TITLE
How do we ingest sysmon logs via osquery?

### DISCUSSION THREAD

**Discussion by greener-dot-ca** [Security-Onion-Solutions/securityonion#1697] [Unsupported Versions] [answered]
I'm aware there's some logic in the osquery packs results parser that will tag sysmon logs for further processing using a sysmon specific pipeline.

What does a pack and queries look like to do this, assuming osquery/launcher is properly configured to access the proper Windows events logs. Is it snapshot or differential? Anything else special to do? What about sysmon configs on the host?

Query something like this?:
```
SELECT * from windows_events WHERE source="Microsoft-Windows-Sysmon/Operational"
```

**Comment by defensivedepth** [comment, marked as answer]
I have added a section in our documentation about this:

https://docs.securityonion.net/en/2.3/osquery.html#shipping-windows-eventlogs

RE: anything special? No - Just install sysmon on the host with whatever config you want and configure the query as specified in the docs - keep in mind it could take up to 15min to start shipping the logs, as osquery checks in with Fleet every 10min to see if there is a config change (ie new scheduled query)
